sharktank/sharktank/utils/export_artifacts.py

```diff
@@ -313,8 +408,8 @@ def compile_to_vmfb(
                 f"--iree-hal-dump-executable-files-to={hal_dump_path}/files"
             ]
         # Append optional arguments if provided
-        if args:
-            compile_args += args
+        if extra_args:
+            compile_args += extra_args
         else:
             compile_args += [
                 "--iree-opt-level=O3",
```

Comment:
This should not be called `extra_args` if it causes to override the default args.
Ideally, we should be able to toggle all options multiple times with the last flag superseding previous flags. I hope IREE does this right like other compilers.
I think we should leave it being `extra_args` and then not override.

Reply:
I will check if we can provide a flag multiple times and change this.